demos/kubernetes/README.md

```diff
@@ -19,17 +19,39 @@ This will deploy two pods, each running a simple echo server and client, and wil
 
 **Send some traffic**
 
-Note the pod corresponding to the apps "echo" and "logic"
+Note the pod corresponding to the apps "echo" and "logic":
     
     kubectl get pods -n <ns>
 
 
-Send HTTP request from "echo" pod to "logic" service
+Send HTTP request from "echo" pod to "logic" service:
 
-    kubectl exec <echo-pod> -c app -n <ns> /bin/client http://logic/<some-text>
+    kubectl exec <echo-pod> -c app -n <ns> /bin/client http://logic/<some-text> -- --count 10
     
-Send HTTP request from "logic" pod to "echo" service
+Send HTTP request from "logic" pod to "echo" service:
 
-    kubectl exec -it <logic-pod> -c app -n demo /bin/client http://echo/<some-text>
+    kubectl exec -it <logic-pod> -c app -n demo /bin/client http://echo/<some-text> -- --count 10
 
 This will echo the URL and print HTTP headers, including "X-Envoy-Expected-Rq-Timeout-Ms".
+
+**Optional - Monitoring with Prometheus and Grafana**
+
+    kubectl apply -f ./prometheus.yaml -n <ns>    
+
+    kubectl apply -f ./grafana.yaml -n <ns>   
+
+Grafana custom image contains a build-in Istio-dashboard that you can access from:
+    
+    http://<grafana-svc-external-IP>:3000/dashboard/db/istio-dashboard
```

Comment:
The only reason for which I used LoadBalancer is because the built-in istio-dashboard in grafana assumes direct access and not via the proxy. I added clarification in the readme file.